src/goose.rs

```diff
@@ -1379,13 +1431,18 @@ impl GooseUser {
     ///     let mut task = task!(loadtest_index_page);
     ///
     ///     async fn loadtest_index_page(user: &GooseUser) {
-    ///         let mut goose = user.get_named("/", "index").await;
-    ///         // Extract the response Result.
+    ///         let mut goose = match user.get_named("/", "index").await {
+    ///             // Return early if get fails, there's nothing else to do.
+    ///             Err(_) => return,
+    ///             // Otherwise unwrap the Result.
+    ///             Ok(g) => g,
+    ///         };
+    ///
     ///         match goose.response {
-    ///             Ok(r) => {
+    ///             Ok(response) => {
     ///                 // We only need to check pages that returned a success status code.
-    ///                 if r.status().is_success() {
-    ///                     match r.text().await {
+    ///                 if response.status().is_success() {
+    ///                     match response.text().await {
```

Comment:
There is enough other changes that warrants this rename to be made here, but strictly speaking this is unrelated, too. ;)